policy/monocamMapper.paf

```diff
@@ -0,0 +1,439 @@
+#<?cfg paf policy ?>
+
+defects:    "../description/defects"
+
+needCalibRegistry: false
+
+levels: {
+    tract: "patch"
+    visit: "channel"
+}
+defaultLevel: "visit"
+defaultSubLevels: {
+    visit: "channel"
+}
+
+exposures: {
+    raw: {
+        template:    "raw/raw_v%(visit)d_f%(filter)s.fits.gz"
+        python:     "lsst.afw.image.ExposureU"
+        persistable: "ExposureU"
+        storage:     "FitsStorage"
+        level:        "Ccd"
+        tables:        raw
+    }
+    raw_amp: {
+        template:    "raw/raw_v%(visit)d_f%(filter)s.fits.gz[%(channel)d]"
+        python:     "lsst.afw.image.ExposureU"
+        persistable:         "ExposureU"
+        storage:     "FitsStorage"
+        level:        "Amp"
+        tables:        raw
+    }
+    postISRCCD: {
+        template:    "postISRCCD/postISRCCD_v%(visit)d_f%(filter)s.fits"
+        python:        "lsst.afw.image.ExposureF"
+        persistable:        "ExposureF"
+        storage:    "FitsStorage"
+        level:        "Ccd"
+        tables:        raw
+    }
+    crDiffim: {
+        template:    "crDiffim/crDiffim_v%(visit)d_f%(filter)s.fits"
+        python:        "lsst.afw.image.ExposureF"
+        persistable:        "ExposureF"
+        storage:    "FitsStorage"
+        level:        "Ccd"
+        tables:        raw
+    }
+    visitim: {
```

Comment:
O.K. thanks.  I obviously cargo culted this and I tried to clean it up a bit, but I appreciate the closer look.